### PR TITLE
Modifiedstory

### PR DESCRIPTION
**Pull request by phantomcoder1996**
Added the chocolate pool story

### CODE REVIEW

**Comment by phantomcoder1996** [comment]
Please review this, @casey-collab

File: english/chocopool/choco.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+  Now you are in the chocolate pool, you are wondering if there are any 
+living creatures in that pool. You keep swimming and swimming.However,you 
+can't find out how deep that pool is.The pool is so wide that you can't
+even guess its width or its length. Suddenly, you hear a strange voice.
+You drink some chocolate to keep you calm, but you can't stop trembling.
+what do you think about this voice??
+
+1.Is it a [bananaboat](bananaboat/bboat.md) floating on the pool surface?
+
+2.Is it the sound of the [hazelnults](hazelnut/hazel.md) terittory?
```
Comment (casey-collab) [inline english/chocopool/choco.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/chocopool/hazelnuts/hazel.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+You have heard alot about the notorious hazelnuts who usually live in
```
Comment (casey-collab) [inline english/chocopool/hazelnuts/hazel.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by phantomcoder1996** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :sunny:

**Pull request merged**